apps/explorer/src/ui/DescriptionList.tsx

```diff
@@ -32,21 +33,37 @@ const descriptionItemLabelStyles = cva(
 	},
 );
 
+const descriptionItemDescriptionStyles = cva(['ml-0 min-w-0 flex-1 leading-none'], {
+	variants: {
+		descriptionJustify: {
```

Comment:
Really don't like this prop name, I'm guessing there's a more descriptive name that would fit better